### PR TITLE
removing hardcoded authentication url

### PR DESCRIPTION
**Pull request by DaJaffaMan**

### CODE REVIEW

**Comment by DaJaffaMan** [comment]
Forgot to mention, this is a working progress will drop another comment when I have a chance to complete.

**Comment by DaJaffaMan** [comment]
This is now ready to be reviewed.

**Review by benjefferies — APPROVED** [review]
Nice, really nice touch with the pop up. Can you squash the commits down to 1 and I will merge?

File: app/src/main/java/uk/echosoft/garageopener/SettingsActivity.java [hunk 1]
```diff
@@ -4,8 +4,12 @@
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.support.v7.widget.Toolbar;
+import android.text.Editable;
```
Comment (benjefferies) [inline app/src/main/java/uk/echosoft/garageopener/SettingsActivity.java]: Can you remove and unused imports before I merge?

**Comment by DaJaffaMan** [comment]
Yes, happy to apply both comments. Will complete tomorrow.

**Comment by DaJaffaMan** [comment]
Commits have been squashed, and imports removed. I'm happy for it to be merged if you are.

**Comment by benjefferies** [comment]
Looks good 👍

**Pull request merged**